app/models/topical_event.rb

```diff
@@ -1,15 +1,23 @@
 class TopicalEvent
-  attr_reader :content_item
+  attr_reader :content_item, :publications, :consultations, :announcements
```

Comment:
Is there a reason we need to use `attr_reader` for these three things, but for the other methods we access them directly?

Reply:
So one reason for having them as currently is to cache the results of getting each document type since the method actually does the database call, as we need to access them twice in the views (once for determining whether we even display a documents section, once for rendering the partial). So didn't want to access the method directly. 

Does that answer your question or am I misunderstanding?

Reply:
Cool, thanks for clarifying.

It's also possible to replicate the same behaviour by memoizing within the method itself, e.g.
```ruby
  def publications
    @publications ||= format_results(fetch_related_documents_with_format("publication"))
  end
```

Reply:
It looks like the memoization within the method is used quite widely within this app, so I'd probably go with that to retain consistency.